src/axiosHelpers/axiosDeleteHelper.ts

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { AxiosResponse } from "../app/models/AxiosResponse";
+import { ACCESS_TOKEN, BASE_URL } from "../constants";
+
+let config = {
+  headers: {
+    "Accept": "application/json",
+    "Content-Type": "application/json",
+    "Access-Control-Allow-Origin": "*",
+    "access-control-allow-methods":"GET, POST, DELETE, PUT, OPTIONS, PATCH, HEAD",
+    "Authorization": `Bearer ${ACCESS_TOKEN}`
+  }
+}
+
+export const axiosDelete = async (url: string): Promise<AxiosResponse> => {
```

Comment:
Continue adjusting all axiosHelpers to pass type. 

`export const axiosDelete = async (url: string): Promise<AxiosResponse<User>> => {
  let result = {} as AxiosResponse<User>;`